src/port1.0/portlivecheck.tcl

```diff
@@ -56,11 +56,13 @@ default livecheck.name default
 default livecheck.distname default
 default livecheck.version {$version}
 default livecheck.ignore_sslcert no
+default livecheck.compression yes
```

Comment:
Adding an option for this may be overkill since livecheck will always want decompressed text to search. I guess it doesn't hurt though.